joomla/joomla-cms
# Make sure when we have no update and no download url we show the correct message

Pull Request for Issue raised by @franz-wohlkoenig 

### Summary of Changes

Make sure when we have no update and no download url we show the correct message

### Testing Instructions

Install 3.9.12-dev
switch to the default update server (which does not know anything about 3.9.12 yet)
Joomla claims it found an update but could not found an download URL wich is not true
apply this patch
notice the correct message comes up
switch to the nighlty update server: https://update.joomla.org/core/nightlies/next_patch_list.xml (which knows of the existence of the 3.9.12-dev)
notice the reinstall message comes up

### Expected result

![image](https://user-images.githubusercontent.com/2596554/63975260-e49b1600-caae-11e9-937d-e15891556a90.png)


### Actual result

![image](https://user-images.githubusercontent.com/2596554/63975238-d8af5400-caae-11e9-89ae-3ce6814b58d6.png)


### Documentation Changes Required

none.
